pkg/client/caller.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"luis/goapi/pkg/entity"
+	"net/http"
+)
+
+const API_ENDPOINT = "https://api.coinbase.com/v2/currencies"
```

Comment:
Good use of global constant, just one thing should this be exported (cause it is on uppercase)?